pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -132,7 +138,7 @@ func (m *Check) emitSysprobeMetrics(snd sender.Sender) error {
 	for _, entry := range stats.Metrics {
 		key := entry.Key
 		metrics := entry.UtilizationMetrics
-		tags := getTagsForKey(key)
+		tags := m.getTagsForKey(key)
```

Comment:
nit: can we rename the receiver (Check) to `c`? (in all struct methods)

Reply:
Done!